src/common/blib.c

```diff
@@ -2689,9 +2689,9 @@ void cmd_exprseq(void) {
     if (count > 1) {
       v_toarray1(var_p, count);
       dx = (xmax - xmin) / (count - 1);
-
+	  x = xmin;
```

Comment:
can you please use the correct indentation with spaces, no tabs.

Reply:
Thanks for adding this. I can see `x` was the wrong datatype.

Could you please add a test for this under samples/distro-examples/tests

You could add this to all.bas with something like this:

```
def expression(x) = x * 10
...

print "EXPRSEQ:" :EXPRSEQ aa, 1, 10, 5 USE expression(x): if aa != [10,32.5,55,77.5,100] then throw "EXPRSEQ failed"

```
You would also need to update samples/distro-examples/tests/output/all.out as PROGLINE and TROFF will have increased by 3 lines.

To run the tests use 

```
make test
make leak-check
```

Cheers,
Chris

Reply:
Ok, I will do.

Reply:
I removed the spaces, added a test for EXPRSEQ and several other functions. "Make test" and "make leak-test" don't report any fails.